src/views/arrange/index.js

```diff
@@ -117,39 +121,104 @@ export class Arrange extends Component {
     ) */
   }
 
+  onDragEnd = result => {
+    const { source, destination } = result
+    if (!destination) {
+      return
+    }
+    
+    if (source.droppableId === destination.droppableId) {

```

Comment:
can you comment what happens in each of these if else block?

Reply:
sure.

Reply:
Yes, I have added the comments to each if else block.